src/main/java/core/basesyntax/service/ProcessTransactionService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.TransactionStrategy;
+
+public interface ProcessTransactionService {
+    Fruit addDataIntoStorage(FruitTransaction fruitTransaction,
+                             StorageDao storageDao,
+                             TransactionStrategy transactionStrategy);
```

Comment:
```suggestion
    Fruit addDataIntoStorage(List<FruitTransaction> fruitTransactions);
```
storageDao and transactionStrategy should be class variables. Service can process whole list of transactions and client won't need to write for loop